helpers/currency.js

```diff
@@ -0,0 +1,7 @@
+export const parsePrice = price =>
+  price.toLocaleString(
+    'pt-BR', {
+      style: 'currency',
+      currency: 'BRL',
+      maximumFractionDigits: 2,
+  })
```

Comment:
Acho que como tu abriu o paranteses em uma identação de `2` espaços e a chaves com uma identação de `4`, melhor fechar a chaves em `4` e o parenteses em `2`... Que tu acha?

```suggestion
    }
  )
```